client/src/Components/Common/Button/index.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
```

Comment:
// fileA.js
export default 1;

export const B = 2;
export const C = 3;

// fileB.js
import * as fileA from './fileA';

console.log(fileA); // { B: 2, C: 3 };

// fileC.js
import fileA from './fileA';

console.log(fileA); // 1 (Default export)


You're not importing React here, you're importing all React's modules as a variable called React. The correct way is to say `import React from 'react';`

I believe the reason why this isn't throwing an error, is because of a somewhat recent update to cra which doesn't necessarily require react to be imported when JSX exists.